src/components/admin/PendingAccounts.vue

```diff
@@ -0,0 +1,76 @@
+<template>
+  <div class="container mt-5">
+    <div class="columns is-tablet">
+      <div class="column">
+        <div
+          class="accounts card custom-card custom-card-padding"
+        >
+          <h2 class="custom-card-title">Comptes en attente de validation</h2>
+          <table class="table is-striped is-fullwidth">
```

Comment:
WE could create a new PR after to improve the loading for all list ?:
- Add spinner 
- Test if we have record, if not, add a text 'no record"